devtools/gradle/gradle-model/src/main/java/io/quarkus/gradle/tooling/GradleApplicationModelBuilder.java

```diff
@@ -167,46 +159,6 @@ private static void collectDestinationDirs(Collection<SourceDir> sources, final
         }
     }
 
-    private PlatformImports resolvePlatformImports(Project project,
```

Comment:
I found it hard for me to parse the separation of concerns between ToolingUtils, GradleApplicationModelBuilder, and ApplicationDeploymentClasspathBuilder to be a bit muddy. Hopefully this consolidation of Configuration-constructing logic into the ApplicationDeploymentClasspathBuilder makes sense and doesn't deviate from the intention.